src/reaper_osara.cpp

```diff
@@ -3389,7 +3389,10 @@ void cmdSplitItems(Command* command) {
 void cmdPaste(Command* command) {
 	MediaItem* item = GetSelectedMediaItem(0, 0);
 	int oldTakes = CountTakes(item);
-	int oldItems = CountMediaItems(0);
+	set<MediaItem*> oldItems;
+	for (int i = 0; i < CountMediaItems(0); i++) {
+		oldItems.insert(GetMediaItem(0, i));
```

Comment:
I worry that this (and the loop below) could cause performance problems in really large projects. This isn't just walking selected media items or items on the current track; it's walking through every item in the entire project. While I understand why that is the case, walking through potentially thousands of items twice every time one pastes seems like it might become problematic. It's not necessarily a show stopper, but I'd like some assurance that this has been verified to be snappy in a massive project before we consider merging.